.envrc.example

```diff
@@ -0,0 +1,19 @@
+toplevel="$(git rev-parse --show-toplevel)"
+
+# ensure the pre-commit git hook is in place
+pushd $toplevel/.git/hooks >/dev/null &&
+    ln -sf ../../.githooks/pre-commit &&
+    popd >/dev/null
+
+# build the managed go sdk
+pushd $toplevel >/dev/null &&
+    make go-check &&
+    popd >/dev/null
+
+# detect os and architecture on darwin and linux
+machine="$(go env GOOS)-$(uname -m)"
+
+# export GOROOT and PATH for managed go sdk
+GOROOT="$toplevel/.build/$machine/go/$(cat .go-version)"
```

Comment:
Why is GOROOT required? This is typically deduced by where the go binary resides. If PATH causes the right go binary to be executed, then GOROOT shouldn't be required?
It _might_ be useful however to unset GOROOT....

Reply:
> It might be useful however to unset GOROOT....
I started with unset, but I think I had an issue installing gopls in an unexpected spot, so I set it, but am perfectly happy to move it back to just unset it.

Reply:
Do you have any details on the failure? If not, I'd rather go with the unset route until we do (FYI, our makefile unset's GOROOT)

Reply:
I am aware that the Makefile unsets `GOROOT`. I think I just futzed about until the first thing worked for me, but didn't fully clean it up. I don't actually need `GOROOT`. Updating.